airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -69,6 +69,27 @@ paths:
               schema:
                 type: string
                 description: Connector manifest template string
+  /v1/manifest/resolve:
+    post:
+      summary: Given a JSON manifest, returns a JSON manifest with all of the $refs and $options resolved and flattened
+      operationId: resolveManifest
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ManifestResolveRequestBody"
```

Comment:
nit: The endpoint is called `resolve_manifest`. Should the input be called `ResolveManifestRequestBody` instead of `ManifestResolveRequestBody`?

Reply:
👍 makes sense to me! Updating.